spec/controllers/vacancies_controller_spec.rb

```diff
@@ -60,6 +60,33 @@
           subject
         end
       end
+
+      context 'search audiotor' do
+        let(:params) do
+          {
+            subject: 'foo'
+          }
+        end
+
+        it 'should call the search auditor' do
+          expect(AuditSearchEventJob).to receive(:perform_later)
```

Comment:
You can also assert against the value of the arguments given.  Otherwise we don't have any confidence about the `audit_row` method.

Reply:
Thanks for pointing this out, fixed in 3701a28

Reply:
That assertion still doesn't give any confidence about `total_count` being populated.  I believe we should add something which tests that?  Or remove it if it isn't important.

Reply:
This been resolved in 1fc5397